### ISSUE TITLE
[docs]: Updated samples for Stacked Form

### ISSUE THREAD

**Issue by Abhijit-Revamp** [Stacked-Org/stacked#1077]
### Describe the missing piece of documentation

Stacked forms have improved a lot and so is the base framework. 
Lot of functions are now ~~deprecated~~ and things changed over 2 years. like addition of **StackedView**

The current documentation is very shallow and the sample code (e.g boxtout) is not up-to-date with all change (say _Using Form Viewmodel with Others like Future and Stream_).

Is there any latest tutorial or sample to refer (even a login / signup form will be of great help)

**Comment by FilledStacks** [comment]
Hey, 

The stacked docs has everything you mentioned here. 

- [Forms](https://stacked.filledstacks.com/docs/getting-started/form-basics/)
- [How Stacked State Management works](https://stacked.filledstacks.com/docs/getting-started/how-it-works)
- [All the ViewModels](https://stacked.filledstacks.com/docs/in-depth/viewmodels/)
- [All Dependency Types](https://stacked.filledstacks.com/docs/in-depth/service-locator/)

And a lot of other docs you can find in the left panel

**Comment by Abhijit-Revamp** [comment]
Yes, I did look at all the documents before, however need a good bit of R&D before can be put to use.
I will share some code that I'm building as an example

**Comment by FilledStacks** [comment]
If you can share what you're trying to achieve I can guide you to the right documentation if it's something that stacked provides for you.

**Comment by alexgoldstone** [comment]
The use-case that brought me to this issues is that the documentation covers the use of Forms with Text input fields but I could do with an example of how other input field types can be created.

For example, how would I use a Switch widget for a Boolean input field? If someone can provide an example I can get working I'd be happy to flesh it out and contribute back to the documentation but so far I'm failing.
